subprojects/docs/src/docs/release/notes.md

```diff
@@ -254,37 +270,37 @@ Related issues:
 <a name="jvm"></a>
 ### JVM
 
-#### Added Java 19 Support
+#### Added Support for Java 19
 
 Gradle 7.6 supports compiling, testing and running on Java 19.
 
-#### Introduced strongly-typed `dependencies` block for JVM test suites
+#### Introduced named dependency arguments in the Kotlin DSL for external dependencies
```

Comment:
I think this is the least interesting and should go at the bottom of the JVM section.